analysis/geospatial_check.sql

```diff
@@ -11,10 +11,10 @@ INSERT INTO geospatial_check_table (
 select :'ccp_v' as v, 	 
         jsonb_agg(t) as result
 from (
-    select jsonb_agg(json_build_object('projectid', tmp.projectid, 'magencyacr',tmp.magencyacr, 
+    select jsonb_agg(json_build_object('projectid', tmp.projectid, 'magencyacro',tmp.magencyacro, 
                               'typecatego', tmp.typecatego)) as values, 
                               'projects_not_within_NYC' as field
-	from (SELECT a.projectid, a.magencyacr, a.typecatego
+	from (SELECT a.projectid, a.magencyacro, a.typecatego
```

Comment:
and also the typecategory? Is there something weird going on that is cutting off the last character on your side

Reply:
yes I think so, seems like my postgres is cutting column names. i will check all of them

Reply:
I modified the table name and add the description field, feels like it will be better if we display a more detailed information on the qaqc page.